.travis.yml

```diff
@@ -26,7 +26,12 @@ before_install:
     - pip3 install -r pip_requirements.txt
     - tests/bin/unittest_setup.py
     - mysql -e 'CREATE DATABASE pattoo_unittest;'
-    - setup/pattoo_installation.py install all
+    - wget https://bootstrap.pypa.io/get-pip.py
+    - sudo python3.6 get-pip.py
```

Comment:
What's the purpose of this?

Reply:
When I tried executing the installation script as sudo in travis, the build would fail due to pip being missing, so I had to install pip in travis' sudo environment to allow for the packages to install whenever the script was run as sudo